readium/navigator/src/main/java/org/readium/r2/navigator/epub/EpubSettingsFactory.kt

```diff
@@ -0,0 +1,367 @@
+package org.readium.r2.navigator.epub
+
+import org.readium.r2.navigator.epub.css.Layout
+import org.readium.r2.navigator.settings.*
+import org.readium.r2.shared.ExperimentalReadiumApi
+import org.readium.r2.shared.publication.LocalizedString
+import org.readium.r2.shared.publication.Metadata
+import org.readium.r2.shared.publication.ReadingProgression
+import org.readium.r2.shared.publication.epub.EpubLayout
+import org.readium.r2.shared.publication.presentation.Presentation
+import org.readium.r2.shared.publication.presentation.presentation
+import org.readium.r2.shared.util.Language
+
+@ExperimentalReadiumApi
+internal class EpubSettingsFactory(
+    private val metadata: Metadata = Metadata(localizedTitle = LocalizedString("")),
```

Comment:
This default value seems a bit out of place. Is it for running unit tests?

Reply:
That's a legacy from something else. We'll see if it's necessary for unit tests (not updated yet).